Makefile

```diff
@@ -6,10 +6,9 @@ r-bindings:
 	mkdir -p tmp
 	rm -f tator-openapi-schema.yaml
 	curl -s -L https://www.tatorapp.com/schema > tator-openapi-schema.yaml
-	./codegen.py tator-openapi-schema.yaml
 	docker run -it --rm \
 		-v $(shell pwd):/pwd \
-		-v $(shell pwd)/tmp:/out openapitools/openapi-generator-cli:latest \
+		-v $(shell pwd)/tmp:/out openapitools/openapi-generator-cli:v5.0.0-beta \
```

Comment:
Using v5.0.0 lets us skip the `remove_one_of` python step.

Reply:
Oh man that is awesome! Need to switch to that in tator/tator-py too then.